src/training/exceptionhandling/ExceptionHandling.java

```diff
@@ -0,0 +1,17 @@
+package training.exceptionhandling;
```

Comment:
1. Implement finally block
2. Add examples of custom exception
3. Comments is missing